config/initializers/omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :facebook, ENV['facebook_key'], ENV['facebook_secret'], :image_size => 'large'
```

Comment:
Line is too long. [89/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.